docker-compose/jaeger-docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
 
     jaeger-agent:
       image: jaegertracing/jaeger-agent
-      command: ["--collector.host-port=jaeger-collector:14267"]
+      command: ["--reporter.tchannel.collector.host-port=jaeger-collector:14267"]
```

Comment:
I will try to depreciate in a way that values from old flags would be used and logged a warning. 

I don't want to keep old flags just as noop. Then I think it's better to fail at init time to avoid misconfiguration.